### ISSUE TITLE
csv2: glint: stack trace when trying to upload image

### ISSUE THREAD

**Issue by rseuster** [hep-gc/cloudscheduler#115]
When in "Images", click on "+Upload Image":
```

Request Method: | GET
-- | --
https://csv2.heprc.uvic.ca/images/upload_image/
2.0.6
TypeError
the JSON object must be str, bytes or bytearray, not 'NoneType'
/usr/lib64/python3.6/json/__init__.py in loads, line 348
/usr/bin/python3
3.6.5
['/opt/cloudscheduler/web_frontend/cloudscheduler',  '/usr/lib64/python36.zip',  '/usr/lib64/python3.6',  '/usr/lib64/python3.6/lib-dynload',  '/usr/lib64/python3.6/site-packages',  '/usr/lib/python3.6/site-packages',  '/']
Tue, 4 Dec 2018 14:09:17 -0800

```
and the traceback:
```
Environment:


Request Method: GET
Request URL: https://csv2.heprc.uvic.ca/images/upload_image/

Django Version: 2.0.6
Python Version: 3.6.5
Installed Applications:
['csv2.apps.Csv2Config',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'mathfilters',
 'glintwebui.apps.GlintwebuiConfig']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.RemoteUserMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
```

**Comment by colsond** [comment]
Looks like the upload function was overlooked when we fixed the transfer function a couple weeks back, i'll apply the relevant changes to this function after todays merge.

**Comment by colsond** [comment]
Should be addressed in commit ddcd73932f10d5929289e2517c68cc668fddb7f4, more thorough testing is required.

I've also noticed there is a lot of permissions issues for the image cache folders that are an easy pitfall. I'd like to eventually change the system to remove the celery portion making file permissions easier but that has sort of been a low priority issue.
